launch/client.py

```diff
@@ -451,6 +451,7 @@ def create_model_endpoint(
         endpoint_type: str = "sync",
         post_inference_hooks: Optional[List[PostInferenceHooks]] = None,
         update_if_exists: bool = False,
+        labels: Optional[Dict[str, str]] = None,
```

Comment:
Do we do any sanitization of these labels in the routing layer?

Reply:
Ooo not at the moment - I suppose this means a public user could pass in `team` and `product` labels, which isn't really what we want.